src/main/AndroidManifest.xml

```diff
@@ -1,2 +1,28 @@
-<!-- Change 'example' below to your company name, and delete this comment. -->
-<manifest package="com.mparticle.kits.example"/>
+<manifest package="com.mparticle.kits.skyhook" xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.CHANGE_WIFI_STATE" />
+    <uses-permission android:name="android.permission.ACCESS_WIFI_STATE" />
+    <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
+    <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
+    <uses-permission android:name="android.permission.WAKE_LOCK" />
+    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+    <uses-permission android:name="android.permission.RECEIVE_BOOT_COMPLETED" />
```

Comment:
Yeah. The description is in the Skyhook doc [here](https://resources.skyhookwireless.com/wiki/type/documentation/skyhook-context/android-quickstart-guide/2327102).

I'll add a link in the README.